v22.1/restore.md

```diff
@@ -20,7 +20,7 @@ You can restore:
 ## Considerations
 
 - `RESTORE` cannot restore backups made by newer versions of CockroachDB.
-- `RESTORE` is a blocking statement. To run a restore job asynchronously, use the `DETACHED` option. See [Options](#options) for more usage detail.
+- `RESTORE` is a blocking statement. To run a restore job asynchronously, use the [`DETACHED`](#detached)` option. See [Options](#options) for more usage detail.
```

Comment:
While I think it is great to link to the exact link here, there is already a link to the options table in the very next sentence. Can you remove one of these here? `#detached` is more specific, so might be better to keep that.